pkg/reconciler/revision/resources/deploy_test.go

```diff
@@ -665,6 +684,130 @@ func TestMakePodSpec(t *testing.T) {
 					withEnvVar("SERVING_REQUEST_METRICS_BACKEND", "opencensus"),
 				),
 			}),
+	}, {
+		name: "podInfoFeature Enabled",
+		fc: apicfg.Features{
+			MountPodInfo: apicfg.Enabled,
+		},
+		rev: revision("bar", "foo",
+			withContainers([]corev1.Container{{
+				Name:           servingContainerName,
+				Image:          "busybox",
+				ReadinessProbe: withTCPReadinessProbe(v1.DefaultUserPort),
+			}}),
+			WithContainerStatuses([]v1.ContainerStatus{{
+				ImageDigest: "busybox@sha256:deadbeef",
+			}}),
+		),
+		want: podSpec(
+			[]corev1.Container{
+				servingContainer(func(container *corev1.Container) {
+					container.Image = "busybox@sha256:deadbeef"
+				}),
+				queueContainer(
+					withPodInfoVolumeMount(),
+				),
+			},
+			withAppendedVolumes(varPodInfoVolume),
+		),
+	}, {
+		name: "podInfoFeature Disabled with annotation features.knative.dev/mount-podinfo=enabled",
```

Comment:
I'd drop the annotation key in the name and instead clarify the purpose  - this comment applies to subsequent tests

ie.
```suggestion
		name: "pod info not mounted when feature disabled and annotation present",
```